src/Task/ComposerNormalize.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class ComposerNormalize.
+ */
+class ComposerNormalize extends AbstractExternalTask
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function canRunInContext(ContextInterface $context)
+    {
+        return $context instanceof GitPreCommitContext || $context instanceof RunContext;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'indent_size' => 4,
```

Comment:
The default in the normallize command is that the `indent_*` options are null.
We prefer to stick with the cli defaults of the tool and make them customizable in the parameters.

Reply:
Fair enough. Fixed.